packages/editor-ui/src/event-bus/source-control.ts

```diff
@@ -1,3 +1,10 @@
+import type { EventBus } from 'n8n-design-system/utils';
 import { createEventBus } from 'n8n-design-system/utils';
 
-export const sourceControlEventBus = createEventBus();
+export interface SourceControlEventBusEvents {
+	pull: never;
+}
+
+export type SourceControlEventBus = EventBus<SourceControlEventBusEvents>;
```

Comment:
do we need to export these? I don't see any code using these.

Reply:
No, not necessarily. I added them just in case to make the types comprehensive. I'll remove it